include/libopencm3/stm32/f1/rcc.h

```diff
@@ -57,6 +58,10 @@
 #define RCC_CSR					MMIO32(RCC_BASE + 0x24)
 #define RCC_AHBRSTR				MMIO32(RCC_BASE + 0x28) /*(**)*/
 #define RCC_CFGR2				MMIO32(RCC_BASE + 0x2c) /*(**)*/
+#define RCC_CFGR3				MMIO32(RCC_BASE + 0x30) /*(^^)*/
+#define RCC_CFGR4				MMIO32(RCC_BASE + 0x50) /*(^^)*/
+#define RCC_CFGR5				MMIO32(RCC_BASE + 0x54) /*(^^)*/
+#define RCC_CFGR6				MMIO32(RCC_BASE + 0x5c) /*(^^)*/
```

Comment:
yeah, so, I'm not going personally merge anything that adds f1 registers for a different target and a different family.  There's got to be a better way.

Reply:
ifdef'ing those counts for (slightly) better way ?

Reply:
I'm not sure honestly, the two approaches people have tried to take so far have largely been "copy paste the entire f1 tree" which is a maintenance nightmare, and "ifdef f1 code to be a trashfire supporting everything in the world"   I imagine breaking it all into much smaller pieces so it can be composed more freely is likely the path forwards, but I've not tried (and it's not remotely on my own personal roadmap)